irods_client/collection_operations.py

```diff
@@ -2,27 +2,28 @@
 import json
 
 class Collections:
-    """" 
-    Initializes Collections with a base url. 
-    Token is set to None initially, and updated when setToken() is called in irodsClient.
-    """
+    
     def __init__(self, url_base: str):
+        """" 
+        Initializes Collections with a base url. 
+        Token is set to None initially, and updated when setToken() is called in irodsClient.
+        """
         self.url_base = url_base
         self.token = None
 
 
-    """
-    Creates a new collection.
+    def create(self, lpath: str, create_intermediates: int=0):
+        """
+        Creates a new collection.
 
-    Parameters
-    - lpath: The absolute logical path of the collection to be created.
-    - create_intermediates (optional): Set to 1 to create intermediates, otherwise set to 0. Defaults to 0.
+        Parameters
+        - lpath: The absolute logical path of the collection to be created.
+        - create_intermediates (optional): Set to 1 to create intermediates, otherwise set to 0. Defaults to 0.
 
-    Returns
-    - 2XX: Status code and iRODS response.
-    - Other: Status code and iRODS error information.
-    """
-    def create(self, lpath: str, create_intermediates: int=0):
+        Returns
+        - 2XX: Status code and iRODS response.
+        - Other: Status code and iRODS error information.
```

Comment:
This could be rephrased to say something like:
```
Returns a dict containing the HTTP status code and iRODS response.
The iRODS response is only valid if no error occurred during HTTP communication.
```

That wording could be used for several operations.

Think on it.